Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp

```diff
@@ -67,6 +66,9 @@ namespace Hazel {
 		m_ZoomLevel -= e.GetYOffset() * 0.25f;
 		m_ZoomLevel = std::max(m_ZoomLevel, 0.25f);
 		m_Camera.SetProjection(-m_AspectRatio * m_ZoomLevel, m_AspectRatio * m_ZoomLevel, -m_ZoomLevel, m_ZoomLevel);
+		
+		m_CameraTranslationSpeed = m_ZoomLevel;
```

Comment:
Good update although if you do it this way, the value for m_CameraTranslationSpeed should be calculated in the constructor rather than set to a default value. I made this optimization in my code when following the video and noticed that, if the camera is moved (with WASD) before a scroll event occurs, it moves much faster than if you zoom in then back to the initial zoom level.

Reply:
By calculated, I realised that I do just mean initializing it to m_ZoomLevel in the Ctor initializer list.

Reply:
Yeah, I did notice that it was a problem even before; we set it to 5 in initialization then immediately overwrote it. We should probably set it a multiple of zoom level and some speed that way its independent of zoom level but proportional to it.